### PR TITLE
Installations for mac

### PR DESCRIPTION
**Pull request by jisie**

### CODE REVIEW

File: book-1-carnival-design/chapters/GETTING_STARTED_MAC.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# Installations
+
+## PostgreSQL
+
+### Installing PostgreSQL Server & pgAdmin 4
+
+We will be using PostgreSQL throughout this course for your relational database management system. pgAdmin is the application we will use to manage and interact with out databases.
```
Comment (rawerner) [inline book-1-carnival-design/chapters/GETTING_STARTED_MAC.md:7]: out = our

File: book-1-carnival-design/chapters/GETTING_STARTED_MAC.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+#### Query Editor
+
+Once you have restored the test database, you can see your tables by navigating into the `dvdrental > Schemas > Public > Tables`.
+
+![Accessing the tables](/images/dvdrentals_tables.png)
+
+Now you can try running a query!
+
+Make sure your dvdrental db in selected. In the image below, **1** opens your query editor. Type in the following:
```
Comment (rawerner) [inline book-1-carnival-design/chapters/GETTING_STARTED_MAC.md:25]: db in selected = is

File: book-1-carnival-design/chapters/GETTING_STARTED_MAC.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+While you have your terminal open, enter in the following command.
+
+```sh
+defaults write com.apple.finder AppleShowAllFiles YES
+```
+
+## SSH Key
+
+SSH is a technology that allows you to create a very secure connection between your computer, and a computer located somewhere else in the world. It's an acronym for Secure SHell. When you create an SSH key on your computer, it actually creates two files
```
Comment (rawerner) [inline book-1-carnival-design/chapters/GETTING_STARTED_MAC.md:89]: SHell

File: book-1-carnival-design/chapters/GETTING_STARTED_MAC.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+1. A public key file that you share with other people and computers. It is usually named `id_rsa.pub`.
+1. A private key file that you never, ever, ever, ever, ever share with anyone. It is usually named `id_rsa`.
+
+### Creating Your SSH Key
+
+Watch a short video for [creating an SSH key in the terminal](https://youtu.be/znRMcNG9_qQ) so that you an work with Github.
```
Comment (rawerner) [inline book-1-carnival-design/chapters/GETTING_STARTED_MAC.md:96]: an = can

File: book-1-carnival-design/chapters/GETTING_STARTED_MAC.md (hunk 1 above)
Comment (rawerner) [inline book-1-carnival-design/chapters/GETTING_STARTED_MAC.md:7]: Could we describe what psql is as well in the paragraph above?

**Comment by jisie** [comment]
@rawerner Hit the up key and forgot to change my commit message, but all changes have been made.

**Pull request merged**